docs/infrastructure.md

```diff
@@ -0,0 +1,82 @@
+# Infrastructure
+
+## Domains
+
+**nextstrain.org** is [hosted on Heroku](#heroku).
+
+**data.nextstrain.org** is an AWS CloudFronted S3 bucket, [`nextstrain-data`](#nextstrain-data).
+
+**staging.nextstrain.org** is an AWS CloudFronted S3 bucket, [`nextstrain-staging`](#nextstrain-staging).
+
+**login.nextstrain.org** is used by our [AWS Cognito user pool](#cognito).
+
+## Heroku
+
+The production Heroku app is [`nextstrain-server`](https://dashboard.heroku.com/apps/nextstrain-server), which is part of a [Heroku app pipeline of the same name](https://dashboard.heroku.com/pipelines/38f67fc7-d93c-40c6-a182-501da2f89d9d).
+Deploys of `master` happen automatically after Travis CI tests are successful.
+
+A testing/staging app, `nextstrain-dev`, is also used.
```

Comment:
Now that we have "review apps" (which look wonderful), how should we be using the testing/staging app?

Reply:
However we see fit! It may turn out not to be used much in practice, in which case we can remove it.